typescript/cli/src/submit/submit.ts

```diff
@@ -81,6 +84,6 @@ async function getTransformer<TProtocol extends ProtocolType>(
         transformerMetadata.props,
       );
     default:
-      throw new Error(`Invalid TxTransformerType: ${transformerMetadata.type}`);
+      throw new Error(`Invalid TxTransformerType.`);
```

Comment:
nit: only use graves when templating

Reply:
TIL grave is an alternative name for backtick :)